packages/angular/builders.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "@angular-devkit/architect/src/builders-schema.json",
+  "builders": {
+    "library-build": {
```

Comment:
We're using `bundle` in other projects. That would be an appropriate name for this as well right?